### DISCUSSION TITLE
Issue with Date Formatting

### DISCUSSION THREAD

**Discussion by muditporwal** [binokochumolvarghese/lightbi-hugo#50] [General]
Hi,

I'm currently experiencing an issue with the date formatting in my  blog. In my `hugo.toml` file located in the project root, I have specified the date format as follows:

```toml
[Params]
selfHosted = true
dateFormat = "2 January 2023"
```

And, when I create a post using Hugo, the date in the front matter appears like this:

```yaml
date = 2024-01-30T21:11:02+08:00
```

But when the post is rendered in the browser, it displays the date as **30 January 21112**(or something similar).

I'm wondering if there's anything else I need to do to ensure proper date formatting. Any guidance or suggestions on resolving this issue would be greatly appreciated.

Thank you!

**Comment by muditporwal** [comment]
I **removed** the dateFormat in the `hugo.toml` so that the code could use the i18 format
```
dateFormat = "2 January 2023"
```

Reference in code : 
```
  {{ $lastmodstr := default (i18n "dateFormat") .Site.Params.dateformat | .Lastmod.Format }}
  {{ $datestr := default (i18n "dateFormat") .Site.Params.dateformat | .Date.Format }}
```

But the example-site is using the dateFormat and I checked it seems to also be having a custom front metadata in the posts : `date = 2024-01-30` but I was not able to get that to work either.

**Comment by binokochumolvarghese** [comment]
I have updated the demo with the latest one. Can you please check demo code [here](https://github.com/binokochumolvarghese/lightbi-hugo-demo).
